### ISSUE TITLE
Code review of ConcentrationDescriber

### ISSUE THREAD

**Issue by zepumph** [phetsims/molarity#144]
As part of https://github.com/phetsims/molarity/issues/141, I was looking in ConcentrationDescriber and had a few thoughts. In general things are looking very good though.

- [x] `isNewSaturationState` has documentation that says "getter for . . .". I would say the normal pattern for this would be to change the private var name to prefix with an underscore, like `this._saturationStateChanged`. Then actually make this method a getter, like:
  > `get saturationStateChanged() { return this._saturationStateChanged; }`
  
  `KeyboardDragListener` can be a good example for this.

- [x] `isNoSolute` is slightly more confusing for this  to return a boolean and have a negative in it. Instead perhaps name it `hasSolute`, and then swap all the usages of it with a `!`.

- [x] `concentrationToIndex` needs doc explaining `maxConcentration`.

>// update properties (documented above) when ConcentrationProperty changes
- [x]  I think that you have this one correct, but I want to mention it to double check. Regarding "properties," since we have `Property.js`, and javascript has `properties` (thus "property" is very overloaded in PhET code, when writing about these in plain English, we make sure to capitalize the word when talking about AXON/Property, and keep it lowercase when it is an object field. In fact, I have taken to just using `field` to describe these member variables quite often, just because I think it is a bit easier. It is totally up to you what you want to do though.

>      // update properties (documented above) when ConcentrationProperty changes
- [x] Second usage of this doc, seems like a copy paste error.

- [x] `getCurrentConcentrationRange` `getCurrentConcentration` and `getCurrentPassiveConcentration` all return strings (at least that is what the doc says), this is not clear from their names, in fact it is confusing because `Range` is a type (thus you could assume it would return a `Range`, and there is another method, `getCurrentSaturatedConcentration`, that returns a number. Please clear up this naming. Glancing at `PositionDescriber.js, it looks like I suffixed with `Clause`, `Region`, `Landmark`, `Sentence` quite often. Please talk to designers that know more about the specifics of this sim for the difference between "region" and "landmark", as there is a difference, and I think this would be a good place to align terminology between the sides of the project.

- [x] `solidsRegion` is of type number, I would call it `solidsIndex` because a "region" to me is the actual string value. Furthermore, its initial value comes from the `solidsToIndex` method. What do you think? 

Again good work, this file is really coming along!

**Comment by twant** [comment]
Just pushed changes, assigning back over for review!

**Comment by zepumph** [comment]
I compared master to the commit before yours, and everything looks really nice. Thanks!

**Comment by twant** [comment]
Thanks @zepumph! Closing this issue.
